services/tracing/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+	"sensorbucket.nl/sensorbucket/internal/env"
+	"sensorbucket.nl/sensorbucket/pkg/mq"
+	"sensorbucket.nl/sensorbucket/services/tracing/migrations"
+	"sensorbucket.nl/sensorbucket/services/tracing/tracing"
+	tracinginfra "sensorbucket.nl/sensorbucket/services/tracing/tracing/infra"
+	tracingtransport "sensorbucket.nl/sensorbucket/services/tracing/tracing/transport"
+)
+
+var (
+	DB_DSN                      = env.Must("DB_DSN")
+	AMQP_HOST                   = env.Must("AMQP_HOST")
+	AMQP_QUEUE_PIPELINEMESSAGES = env.Must("AMQP_QUEUE_PIPELINEMESSAGES")
+	AMQP_QUEUE_ERRORS           = env.Must("AMQP_QUEUE_ERRORS")
+)
+
+func main() {
+	db, err := createDB()
+	if err != nil {
+		panic(fmt.Sprintf("could not create database connection: %v\n", err))
+	}
+
+	// Setup the tracing service
+	tracingStepStore := tracinginfra.NewPSQL(db)
+	tracingService := tracing.New(tracingStepStore)
+
+	// Setup MQ Transports
+	amqpConn := mq.NewConnection(AMQP_HOST)
+	tracingtransport.StartMQ(tracingService, amqpConn, AMQP_QUEUE_ERRORS, AMQP_QUEUE_PIPELINEMESSAGES)
+	go amqpConn.Start()
+
+	select {}
```

Comment:
What's this?

Reply:
a hack to keep the service running :p. fixed in: PR #30